nuxt.config.js

```diff
@@ -55,7 +55,7 @@ export default {
     ],
     '@nuxtjs/speedcurve',
     '~/modules/nuxt-module-ipfs',
-    ['nuxt-canonical', { baseUrl: 'https://ipfs.io' }],
+    ['nuxt-canonical', { baseUrl: 'https://ipfs.tech' }],
```

Comment:
:information_source: this takes care of [canonical link](https://en.wikipedia.org/wiki/Canonical_link_element) and SEO migration